salt/utils/cloud.py

```diff
@@ -1356,18 +1356,28 @@ def deploy_windows(host,
                 salt.utils.smb.delete_directory('salttemp', 'C$', conn=smb_conn)
         # Shell out to psexec to ensure salt-minion service started
         if use_winrm:
+            
             winrm_cmd(winrm_session, 'sc', ['stop', 'salt-minion'])
-            time.sleep(5)
+            time.sleep(10)
```

Comment:
Ideally we'd come up with a way to check and not need the sleep time, similar to what you are doing with the while loop below.

Reply:
Ideally yes. But not really sure what winrm returns. Don't know how to handle the result / output, so just increased the timeout.

Reply:
A continuation of my [other comments](https://github.com/saltstack/salt/pull/53547#issuecomment-552915073), one of these PRs should be pointed to master, and have tests applied.

If you don't have access to a windows environment to figure out what is returned here, I'm sure we can find someone who can get that information.

Reply:
@waynew I figured it out...
It returns the status code. see:

```
def winrm_cmd(session, command, flags, **kwargs):
    '''
    Wrapper for commands to be run against Windows boxes using WinRM.
    '''
    log.debug('Executing WinRM command: %s %s', command, flags)
    r = session.run_cmd(command, flags)
    return r.status_code
```
Not really sure what to do with it.


What do you mean by "one of these PRs should be pointed to master"?
Should I change the branch and merge to master?

Thanks